filter/src/main/java/gov/cms/ab2d/filter/FilterOutByDate.java

```diff
@@ -39,13 +39,15 @@
          * @throws ParseException if there was an error constructing the Date objects
          */
         public DateRange(Date start, Date end) throws ParseException {
+            SimpleDateFormat fullDateFormat = new SimpleDateFormat(FULL);
```

Comment:
This variable does not appear to be used within the constructor; we can probably drop it.